modules/openapi-generator/src/main/resources/Java/libraries/jersey2/anyof_model.mustache

```diff
@@ -69,6 +86,7 @@ public class {{classname}} extends AbstractOpenApiSchema{{#vendorExtensions.x-im
         schemas.put("{{{.}}}", new GenericType<{{{.}}}>() {
         });
         {{/anyOf}}
+        JSON.registerDescendants({{classname}}.class, Collections.unmodifiableMap(schemas));
```

Comment:
@wing328 , it's a bit hacky for now, I'd rather not use a public method, but it's needed to have proper deserialization when there are multiple levels of oneOf/anyOf.